src/lib/dexrt.cpp

```diff
@@ -161,6 +161,15 @@ void encodePNG(char **resultPtr, int8_t* pixels, int32_t width, int32_t height)
 // The string buffer size used for converting integer and floating-point types.
 static constexpr int showStringBufferSize = 32;
 
+void showNat32(char **resultPtr, uint32_t x) {
```

Comment:
That shouldn't be necessary anymore, right?

Reply:
We still use it in the prelude, just now it's for the `Show Word32` instance. The hack to make `Word32` print like `Nat` is one of the big warts in the shortcut we're taking.